scripts/build/Platform/Mac/build_config.json

```diff
@@ -89,6 +90,22 @@
       "ASSET_PROCESSOR_PLATFORMS": "mac"
     }
   },
+  "test_profile": {
+    "TAGS": [
+        "daily-pipeline-metrics",
+        "weekly-build-metrics"
+    ],
+    "COMMAND": "build_test_mac.sh",
+    "PARAMETERS": {
+      "CONFIGURATION": "profile",
+      "OUTPUT_DIRECTORY": "build/mac",
+      "CMAKE_OPTIONS": "-G Xcode",
+      "CMAKE_LY_PROJECTS": "AutomatedTesting",
+      "CMAKE_TARGET": "ALL_BUILD",
+      "CTEST_OPTIONS": "-E (AutomatedTesting::Atom_TestSuite_Main|AutomatedTesting::TerrainTests_Main|Gem::EMotionFX.Editor.Tests) -L (SUITE_smoke|SUITE_main) -LE (REQUIRES_gpu) --no-tests=error",
```

Comment:
Please remove the exclusion `-E (AutomatedTesting::Atom_TestSuite_Main|AutomatedTesting::TerrainTests_Main|Gem::EMotionFX.Editor.Tests)`.

That was added on Linux only because they fail at the moment and Linux tests gate PRs. That exclusion is temporal and not meant to be copied to other platforms. Since Mac tests will run on nightly build only, there is not reason to exclude them. So please remove the exclusion so we can know if they pass or fail in Mac and act accordingly.

Reply:
sure, removing

Reply:
Awesome, thanks! Approving.

Reply:
`Gem::EMotionFX.Editor.Tests.main::TEST_RUN` was failing from that filter, however, note that pytest is disabled in Mac, so the `AutomatedTesting` tests are not running either

Reply:
>pytest is disabled in Mac

Good, then `AutomatedTesting::Atom_TestSuite_Main` and `AutomatedTesting::TerrainTests_Main` will not run for now and they will once pytest is enabled for mac.

> Gem::EMotionFX.Editor.Tests.main::TEST_RUN was failing from that filter

These are unit tests, so if they fail please create a task for the sig of the animation team to take a look at them. They fail in linux too, but might be for different reasons.

Reply:
Logged:
https://github.com/o3de/o3de/issues/6751
https://github.com/o3de/o3de/issues/6752
https://github.com/o3de/o3de/issues/6753
https://github.com/o3de/o3de/issues/6754

(wanted to wait for the nightly to run to grab the full list)